sbin/build.sh

```diff
@@ -21,6 +21,7 @@ TARGET_DIR=$2
 OPENJDK_REPO_NAME=$3
 BUILD_FULL_NAME=$4
 JVM_VARIANT=${5:=normal}
+NOBUILD=$6
```

Comment:
So previously if you didn't set the `JVM_VARIANT` it would default to `normal`, now that's only the case if `NOBUILD` isn't set? That seems confusing.